bridge-token-factory/src/lib.rs

```diff
@@ -78,6 +81,56 @@ pub enum Role {
     UnrestrictedDeposit,
     UnrestrictedDeployBridgeToken,
     MetadataManager,
+    FeeSetter,
+    FeeClaimer,
+}
+
+#[derive(
+    Default, BorshDeserialize, BorshSerialize, Debug, Clone, Serialize, Deserialize, PartialEq,
+)]
+pub struct DepositTokenBounds {
+    lower_bound: u128,
+    upper_bound: u128,
+}
+
+#[derive(
+    Default, BorshDeserialize, BorshSerialize, Debug, Clone, Serialize, Deserialize, PartialEq,
+)]
+pub struct WithdrawTokenBounds {
```

Comment:
```suggestion
pub struct WithdrawFeeBounds {
```

Reply:
used single struct now, reducing code size. https://github.com/aurora-is-near/rainbow-token-connector/commit/e53607f60125d25cb40abd0482f1120da1552d62